src/app/common/components/MustGatherModal.tsx

```diff
@@ -39,32 +40,31 @@ export const MustGatherModal: React.FunctionComponent = () => {
   );
 
   const handleMustGatherRequest = ({ customName, type }: MustGatherObjType) => {
+    const namespacedName = withNs(customName, type);
     registerMustGather.mutate({
-      'custom-name': customName,
+      'custom-name': namespacedName,
```

Comment:
The fact that `custom-name` gets set to `namespacedName` in the backend, but in the UI `customName` is a different non-namespaced thing, feels a bit awkward to me. We can leave it, but I think we might want to invert it. We could call your current `customName` something like `displayName` or `resourceName` or `planOrVmName` or something, and change your current `namespacedName` to `customName` so it matches what the backend calls `custom-name`?

I'm overthinking it again.

Reply:
It's a good point though. Yea gimme a few and I'll make it more readable